integrationExamples/gpt/idImportLibrary_example.html

```diff
@@ -0,0 +1,351 @@
+<html>
+<head>
+    <script>
+      var pbjs = pbjs || {};
+      pbjs.que = pbjs.que || [];
+    </script>
+    <script src="../../build/dev/prebid.js" async></script>
+
+    <script>
+      var googletag = googletag || {};
+      googletag.cmd = googletag.cmd || [];
+      googletag.cmd.push(function() {
+        googletag.pubads().disableInitialLoad();
+      });
+
+      pbjs.que.push(function() {
+        pbjs.setConfig({
+          debug: true,
+          consentManagement: {
+            cmpApi: 'iab',
+            timeout: 1000,
+            allowAuctionWithoutConsent: true
+          },
+          userSync: {
+            userIds: [
+              {
+                name: "id5Id",
+                params: {
+                  partner: 173 //Set your real ID5 partner ID here for production, please ask for one at http://id5.io/prebid
+                },
+                storage: {
+                  type: "cookie",
```

Comment:
we recommend to our customers to use `html5` now, so could we keep this example file consistent, please? thanks!

```suggestion
                  type: "html5",
```

Reply:
thank you